package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@extole/react-native-sdk",
   "version": "0.0.2",
   "description": "Extole React Native SDK",
-  "main": "lib/commonjs/index",
-  "module": "lib/module/index",
+  "main": "lib/commonjs/index.js",
+  "module": "lib/module/index.js",
   "types": "lib/typescript/index.d.ts",
-  "react-native": "src/index",
+  "react-native": "src/index.tsx",
```

Comment:
can you please explain the current problem?

Reply:
You are absolutely right. These changes are unnecessary. 
The file extensions were included with the default output of `builder bob`; 
![image](https://user-images.githubusercontent.com/11494223/218539920-626ef73e-d135-4120-83aa-0a9168c924f9.png)
I noticed it was outdated, so I updated the dependency and ran to keep the project fresh.